store/pdtypes/statistics.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pdtypes
+
+// RegionStats records a list of regions' statistics and distribution status.
+type RegionStats struct {
```

Comment:
It could be tough to maintain these variables and better to add a reference in both TiDB and PD repos.

Reply:
PD should remain backward compatible at the API level, so there is no need to worry about version updates, and copying the code is a more convenient way to go.